### PR TITLE
Update doc for image-queue and browser test

### PR DESCRIPTION
**Pull request by zhangyiatmicrosoft**
## Launch Checklist

- Update life-of-a-tile.md to reflect image-request change
- Update README.md for browser tests

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [x] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 - Update CONTRIBUTING.md with details on setting up on M1 mac ([#2196](https://github.com/maplibre/maplibre-gl-js/pull/2196))
 - Update default type of originalEvent in MapLibreEvent to be `unknown` ([#2243](https://github.com/maplibre/maplibre-gl-js/pull/2243))
 - Improve performance when forcing full symbol placement by short circuiting pause checks ([#2241](https://github.com/maplibre/maplibre-gl-js/pull/2241))
+- Update README.md for broweser test and life-of-a-tile.md to reflect image-request change ([#2280](https://github.com/maplibre/maplibre-gl-js/pull/2280))
```
Comment (HarelM) [inline CHANGELOG.md:18]: I don't think that is interesting enough to be in the changelog, having said that, feel free to move the entry above the features to the right place...

File: test/integration/browser/README.md [hunk 2]
```diff
@@ -11,8 +11,7 @@ To run Webdriver, you'll have to install the driver for every browser you want t
 
 ## Running
 
-- Run browser tests with `npm run test-browser`.
-- The tests default to Chrome, but it's possible to use a different browser by setting the `SELENIUM_BROWSER` environment variable, e.g. like this: `SELENIUM_BROWSER=firefox npm run test-browser`.
-- To run on iOS Safari, use `SELENIUM_BROWSER=safari::ios npm run test-browser`. Make sure that the iOS device is in the same local Wifi network as your computer.
-- To run on Android Chrome, use `SELENIUM_BROWSER=chrome::android npm run test-browser`. Make sure that the Android device is in the same local Wifi network as your computer.
-- To run individual tests instead of the entire test suite, you can execute a test file with the TAP runner, e.g. by typing `build/run-tap test/browser/zoom.test.js`.
+- Run browser tests with `npx jest test/integration/browser/browser.test.ts`.
+- The tests default to Chrome, but it's possible to use a different browser by setting the `SELENIUM_BROWSER` environment variable, e.g. like this: `SELENIUM_BROWSER=firefox npx jest test/integration/browser/browser.test.ts` (on Linux platform).
```
Comment (HarelM) [inline test/integration/browser/README.md:15]: I don't think this is true anymore, I couldn't find this selenium variable in the test code and I don't think you can use other browsers easily...

Comment (zhangyiatmicrosoft) [reply test/integration/browser/README.md:15]: ah..ok. I will remove them all

File: CHANGELOG.md (hunk 1 above)
Comment (zhangyiatmicrosoft) [reply CHANGELOG.md:18]: reverted

**Review by HarelM — APPROVED** [review]

**Pull request merged**